tests/http/bad-actors.test.js

```diff
@@ -5,11 +5,13 @@ var instance = server({ useDomains: false });
 describe('bad actors', () => {
   it('should not throw error when accessing /����%20wwwroot.rar', (done) => {
     instance.inject({
-      url: '/����%20wwwroot.rar'
+      url: '/����%20wwwroot.rar',
+      headers: {
+        'Accept-Language':"fr,en;q=0.5"
+      }
     }, (response) => {
-      should(response.statusCode).equal(400);
-      should(response.result.message).equal('Location cannot contain or convert into non-ascii characters');
-
+      should(response.statusCode).equal(302);
+      should(response.headers.location).equal('/fr/');
```

Comment:
The route-file-content is handling those 404 now and we don't hit this anymore and its doing a proper redirect now.